src/main/kotlin/racing/domain/RaceProcessor.kt

```diff
@@ -0,0 +1,32 @@
+package racing.domain
+
+class RaceProcessor {
```

Comment:
```suggestion
object class RaceProcessor {
```

RaceProcessor는 프로퍼티를 가지고 있지 않고, 외부에서 주는 값을 통해서만 메서드가 실행되고 있기에 object class로도 사용이 가능해보이네요~